src/test/java/com/theelixrlabs/healthcare/controller/PatientControllerTest.java

```diff
@@ -0,0 +1,90 @@
+package com.theelixrlabs.healthcare.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.theelixrlabs.healthcare.constants.ApiPathsConstant;
+import com.theelixrlabs.healthcare.constants.TestConstants;
+import com.theelixrlabs.healthcare.dto.PatientDto;
+import com.theelixrlabs.healthcare.exceptionHandler.PatientNotFoundException;
+import com.theelixrlabs.healthcare.response.FailureResponse;
+import com.theelixrlabs.healthcare.response.SuccessResponse;
+import com.theelixrlabs.healthcare.service.PatientService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * Contains test methods for Patient Controller
+ * uses mock mvc to simulate Http requests and verify the behaviour of each method
+ */
+@WebMvcTest(PatientController.class)
+public class PatientControllerTest {
+
+    @Autowired
```

Comment:
Instead of Auto wiring try like below

@BeforeMethod
    public void init() {
        MockitoAnnotations.openMocks(this);
        this.mockMvc = MockMvcBuilders.standaloneSetup().build();
    }

Reply:
Resolved